EIPS/eip-725.md

```diff
@@ -1,9 +1,9 @@
 ---
 eip: 725
-title: General key-value store and execution standard
+title: General data key/value store and execution standard
```

Comment:
We're discouraging "standard" in titles and replaced "Simple Summary" with descriptions now:

```suggestion
title: General data key/value store and execution
description: An interface for a smart contract based account with attachable data key/value store
```